webui/react/src/pages/ProjectDetails.tsx

```diff
@@ -47,7 +49,7 @@ const ProjectDetails: React.FC = () => {
   const [canceler] = useState(new AbortController());
   const pageRef = useRef<HTMLElement>(null);
 
-  const [workspace, setWorkspace] = useState<Workspace>();
+  const workspace = useObservable(workspaceStore.getWorkspace(project?.workspaceId));
```

Comment:
**suggestion:** since the only reference to this is expecting a `Workspace` and not needing to know if it's a `Loadable` or not, it might be easier to just make this call:
```
const workspace = Loadable.getOrElse(undefined, useObservable(workspaceStore.getWorkspace(project?.workspaceId)));
```